### PR TITLE
Add implementations to README

### LINKED ISSUE #192: Q. Can I add openwallet foundation implementation for token status list in README?
Hi members of WG.

I'm developing SD-JWT VC library for Typescript in OWF. I'm also developing token status list standard in my library as well.
Here is the link: https://github.com/openwallet-foundation/sd-jwt-js
and You can check the token status list library here: https://github.com/openwallet-foundation/sd-jwt-js/tree/main/packages/jwt-status-list

So I'd like to add a link of my implementation here. Is it okay?

### PR DESCRIPTION
**Pull request by paulbastian**
Closes #192

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ $ make
 Command line usage requires that you have the necessary software installed.  See
 [the instructions](https://github.com/martinthomson/i-d-template/blob/main/doc/SETUP.md).
 
+# Implementations
+
+- TypeScript: [OWF sd-jwt-js](https://github.com/openwallet-foundation/sd-jwt-js/tree/main/packages/jwt-status-list)
```
Comment (lukasjhan) [inline README.md:33]: In my point of view, the repo is would be better: https://github.com/openwallet-foundation/sd-jwt-js
What do you guys think?

Comment (paulbastian) [reply README.md:33]: sure I don't mind

**Review by lukasjhan — APPROVED** [review]
Thank you

**Review by c2bo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -28,6 +28,10 @@ $ make
 Command line usage requires that you have the necessary software installed.  See
 [the instructions](https://github.com/martinthomson/i-d-template/blob/main/doc/SETUP.md).
 
+# Implementations
+
+- TypeScript: [OWF sd-jwt-js](https://github.com/openwallet-foundation/sd-jwt-js)
```
Comment (paulbastian) [inline README.md:33]:
```suggestion
- : 
| Project Name | Link | Language | Issuer | Holder | Relying Party |
|---|---|---|---|---| --- |
| OWF sd-jwt-js | [link](https://github.com/openwallet-foundation/sd-jwt-js) | TypeScript | yes | yes | yes |
```

**Pull request merged**